ADOPTERS.md

```diff
@@ -0,0 +1,19 @@
+# Adopters
+
+You can find here a list of organizations and/or or projects that either use, or have used, the _SAP Project Portal for InnerSource_, either directly or as a reference.
+
+| Organization | Details | References |
+| ------------ | ------- | ---------- |
+|              |         |            |
```

Comment:
Maybe add SAP as the first adopter here?
Having an example of how to fill in this table could be really helpful, in addition to the explanations below.

If you do that, I would also add a row to the table like this:

| Organization | Details | References |
| ------------ | ------- | ---------- |
|  SAP          |  TBD       | TBD           |
|  Your org ...            | could be listed ...        | here! :)           |

Reply:
Interesting, as SAP is providing the project, I did not consider adding it to the list, but that is a good idea, I'll add that.
The other row (Your org ... ) I have mixed feelings, but I can't give a good reason not to take it. I'll incorporate it as well.
Additionally, I realized that it would be better to call the _Contribute_ heading something else - I'm considering _How to add Adopters_ or _How to update the Adopters list_, I'm more inclined to the second, as it refers to the general maintenance of the table (e.g. new references for existing entries), not only adding new entries.

Reply:
Resolved with 94abf9808a71da4eae99eeb10b667757fb9c85ed.